engine/cron_job_declarations.go

```diff
@@ -46,11 +46,6 @@ func (c *Client) cronJobs() taskmanager.CronJobs {
 		60*time.Second,
 		taskCleanupDraftTransactions,
 	)
-	addJob(
```

Comment:
I just realised that you didn't change transaction flow, 
So what actually will happen now if broadcasting will fail during sending and receiving transaction

Reply:
I have to rethink & investigate that. I'm closing this PR for now.
I'll make another one soon.